src/test/pow_tests.cpp

```diff
@@ -44,7 +48,12 @@ BOOST_AUTO_TEST_CASE(get_next_work_lower_limit_actual)
     pindexLast.nHeight = 68543;
     pindexLast.nTime = 1279297671;  // Block #68543
     pindexLast.nBits = 0x1c05a3f4;
-    BOOST_CHECK_EQUAL(CalculateNextWorkRequired(&pindexLast, nLastRetargetTime, chainParams->GetConsensus()), 0x1c0168fdU);
+    unsigned int expected_nbits = 0x1c0168fdU;
```

Comment:
can this relation between `pindexLast.nBits` and `expected_nbits` be explicitly computed? Would make the test condition below abundantly clear.

Reply:
I'm not sure there's a great way to do this without essentially duplicating the code from `pow.cpp`; is that what you have in mind?

Reply:
Not convinced either way, maybe just comment on the relationship here?

Reply:
Added a comment, let me know if this is what you had in mind (or please suggest some other language?)